lib/rmg/CMapGenerator.cpp

```diff
@@ -275,13 +274,13 @@ void CMapGenerator::genZones()
 	logGlobal->info("Zones generated successfully");
 }
 
-void CMapGenerator::createWaterTreasures()
+void CMapGenerator::createWaterTreasures() const
```

Comment:
Another questionable const addition.
Yes, state of object itself remains the same, but internal state changes.
Probably should not be marked as const

Reply:
removed